scripts/build/Dockerfile.archlinux

```diff
@@ -0,0 +1,43 @@
+FROM archlinux
+ARG CC=gcc
+ARG ENV1=FOOBAR
+
+RUN pacman -Syu --noconfirm \
+	$CC \
+	bash \
+	make \
+	ccache \
```

Comment:
The ccache setup was removed in https://github.com/checkpoint-restore/criu/pull/1395